src/logic/operate.js

```diff
@@ -0,0 +1,20 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  let result;
+  const firstNumber = Big(numberOne);
+  const secondNumber = Big(numberTwo);
+
+  if (operation === '+') {
+    result = firstNumber.plus(secondNumber);
+  } else if (operation === '-') {
+    result = firstNumber.minus(secondNumber);
+  } else if (operation === 'x') {
+    result = firstNumber.times(secondNumber);
+  } else if (operation === '÷') {
+    result = firstNumber.div(secondNumber);
+  } else if (operation === '%') {
```

Comment:
### [Optional]
It is a good practice to use `switch case` once our condition is more than 3;
https://medium.com/the-daily-standup/javascript-switch-statement-vs-if-else-if-ee3111793fa0